content/rest/messages.textile

```diff
@@ -157,6 +157,10 @@ bc[swift]. channel.publish("event", data: "This is my payload")
 
 bc[go]. channel.Publish("event", "This is my payload")
 
+h4(#idempotent). Enabling idempotent publishing
+
+It is possible for a client publishing through REST to not receive an acknowledgement of receipt from Ably for numerous reasons, which can lead to the client re-publishing a message. If you wish to ensure that Ably can identify incorrectly re-sent messages, you should "enable idempotent publishing":/rest/usage#client-options. This attaches an identifier to each message sent from the client, enabling Ably to detect duplicates of the same message and ignore them.
```

Comment:
> identify incorrectly re-sent messages
they're not incorrectly re-sent. Perhaps this should say "If you wish to ensure that Ably can recognise these re-sent messages as duplicates, ..."